tsconfig.base.json

```
@@ -119,6 +119,9 @@
       "@genshin-optimizer/sr/db": ["libs/sr/db/src/index.ts"],
       "@genshin-optimizer/sr/dm": ["libs/sr/dm/src/index.ts"],
       "@genshin-optimizer/sr/formula": ["libs/sr/formula/src/index.ts"],
+      "@genshin-optimizer/sr/page-lightcones": [
+        "libs/sr/page-lightcones/src/index.ts"
```

```suggestion
        "libs/sr/page-lightcones/src/index.tsx"
```

This will fix the build issue. I also had the same thing, it took a while to debug I'm not sure why the error is so unclear